### DISCUSSION TITLE
Do we want to create a development branch?🌳

### DISCUSSION THREAD

**Discussion by uo289267** [Arquisoft/wiq_en1b#37] [Ideas]
A development branch would allow us to create pull requests to it and to make sure everything works as expected before merging with master. It is a safe way of developing without risking breaking master with some mistake.

**Comment by Mister-Mario** [comment]
I also like the idea but I think Pablo needs to approve this

**Comment by jcano897** [comment]
Yeah I think is a great idea, specially now that we removed the unnecessary branches
